webrtc-stats.html

```diff
@@ -2953,13 +2953,13 @@ <h2>
                 </div>
               </dd>
               <dt>
-                <dfn><code>ip</code></dfn> of type <span class=
+                <dfn><code>address</code></dfn> of type <span class=
                 "idlMemberType"><a>DOMString</a></span>
               </dt>
               <dd>
                 <p>
-                  It is the IP address of the candidate, allowing for IPv4 addresses and IPv6
-                  addresses, but fully qualified domain names (FQDNs) are not allowed. See
+                  It is the IP address of the candidate, allowing for IPv4 addresses, IPv6
```

Comment:
is the address of the candidate

Reply:
oh that is the prose from 5245... which no longer seems to be in the successor? anyway, :+1: on using that prose for consistency

Reply:
As discussed offline, this matches the language in [rfc5245](https://tools.ietf.org/html/rfc5245#section-15.1). I suppose it has a bit of a terminology issue. Anyone feel strongly which way it should be here?

Reply:
Turns out others agree with you, so I made the change. Thanks!